### DISCUSSION TITLE
Update GNOME Software (and maybe GNOME Web) icon based on new upstream

### DISCUSSION THREAD

**Discussion by vitalkanev** [ubuntu/yaru#2789] [General]
I am not using a template so people could discuss stuff I think.

Icon name | Current Yaru icons | New upstream
----|----|----
`webbrowser-app` | ![YaruBrowser](https://user-images.githubusercontent.com/18398761/112866844-eda4fc80-90c2-11eb-9641-8d1c17910acd.png) | ![NuBrowser](https://user-images.githubusercontent.com/18398761/112866926-031a2680-90c3-11eb-8b15-878dcb25e175.png)
`org.gnome.Software` | ![YaruSoftware](https://user-images.githubusercontent.com/18398761/112869621-09f66880-90c6-11eb-9a87-b6e810a8a2df.png) | ![NuSoftware](https://user-images.githubusercontent.com/18398761/112869623-0a8eff00-90c6-11eb-9de4-7924f55c266f.png)

What do you think: should we make new version of above icons? Or do something else?

**Comment by Feichtmeier** [comment]
Since those apps are not pre-installed they are not so super important.
But currently we have a nice set of yaru icons for all core gnome apps except those gnome designers explicetly asked us to remove (podcast app for example :shrug: )
Anyways, since the current ones are fine, and we got much to do anyways, I would rather categorize this as low prio or eventually not do this at all. However, if our icon designers are interested they could eventually create something different here if it makes sense.
Looking at the browser icon, I really think the current classy yaru navigator icon is a pretty damn solid icon. Mimicking upstream without thinking about it is not my cup of tea :tea: :)
The software icon could eventually be yarufied though.

**Comment by clobrano** [comment]
What do you think about dropping the "status: discussion" tag and just converting the bugs into github's discussions?

**Comment by Feichtmeier** [comment]
sounds very good!

**Comment by Benni3-1415** [comment]
I think the _GNOME Web_-icon (Epiphany) [looks like Safari](https://www.omgubuntu.co.uk/2016/06/ubuntu-browser-icon-too-similar-safari "omg! ubuntu!").

### Some quick concept designs
| As background texture | As wires | As faces | With magnifying glass |
|:---:|:---:|:---:|:---:|
| ![Background lines](https://user-images.githubusercontent.com/73243667/120925408-ca0cbc80-c6d8-11eb-9df4-631cd7b069be.png) |  ![lines-cut](https://user-images.githubusercontent.com/73243667/120925639-bf9ef280-c6d9-11eb-9a9f-fea436a17448.png) <br> ![lines](https://user-images.githubusercontent.com/73243667/120925679-f1b05480-c6d9-11eb-8dfd-19060abf6aa8.png) | ![faces](https://user-images.githubusercontent.com/73243667/120925602-8f575400-c6d9-11eb-8f4d-9fa5b049dafc.png) | ![lines-cur magnify-large](https://user-images.githubusercontent.com/73243667/120925470-1d7f0a80-c6d9-11eb-82c6-ffebb003ed02.png) <br> ![lines-cut magnify](https://user-images.githubusercontent.com/73243667/120925543-4dc6a900-c6d9-11eb-9c9d-1cde7ba29616.png) |
| :confused: | :star: 2th favorite (first one) | looks like a ball | :star::star: My favorite (first one) |

I would create a PR for it.

**Reply by Benni3-1415** [reply]
@clobrano @Feichtmeier, wich icon is the best?

**Reply by Feichtmeier** [reply]
Seriously, I would prefer to keep the current icon :shrug:
